src/sentry/utils/options.py

```diff
@@ -0,0 +1,28 @@
+""" Helper functions for sentry.options """
+
+
+import logging
+
+from sentry import options
+
+logger = logging.getLogger(__name__)
+
+
+def sample_modulo(option_name: str, value: int, granularity: int = 100) -> bool:
+    """Deterministically take a sampling decision given an integer value.
+
+    Assumes that
+
+    * `option_name` is a registered option,
+    * `(value % granularity)` is uniformly distributed (e.g. org ID, project ID, ...).
+
+    Inspired by https://github.com/getsentry/snuba/blob/28891df3665989ec10e051362dbb84f94aea2f1a/snuba/state/__init__.py#L397-L401
+    """
+    option_value = options.get(option_name)
+    try:
+        if (value % granularity) < granularity * option_value:
```

Comment:
nit: I'd use another name that isn't that similar to `value`, it might be confusing when reading this last line
```suggestion
    sample_rate = options.get(option_name)
    try:
        if (value % granularity) < sample_rate * granularity:
```
What this function does is to sample, so I suggest `sampling_rate`.

Reply:
Good idea!